pyteal/compiler/compiler.py

```diff
@@ -181,6 +187,7 @@ def compileTeal(
             constants will be assembled in the most space-efficient way, so enabling this may reduce
             the compiled program's size. Enabling this option requires a minimum TEAL version of 3.
             Defaults to false.
+        optimize (optional): todo
```

Comment:
@algoidurovic Friendly visual reminder to fill in the TODO.

Reply:
good catch, fixed.